app/views/planning_application/assessment_tasks/_check_consistency.html.erb

```diff
@@ -17,12 +17,20 @@
         )
       ) %>
     </li>
-    <% if @planning_application.planning_history_enabled? %>
-      <li class="app-task-list__item">
+    <li class="app-task-list__item">
+      <% if @planning_application.planning_history_enabled? %>
         <span class="app-task-list__task-name">
-          <%= link_to "History", planning_application_planning_history_path(@planning_application), class: "govuk-link" %>
+          <%= link_to(
+            t(".history"),
+            planning_application_planning_history_path(@planning_application),
+            class: "govuk-link"
+          ) %>
         </span>
-      </li>
-    <% end %>
+      <% else %>
+        <%= @planning_application.assessment_detail_tasklist(
+          :past_applications
+        ) %>
+      <% end %>
```

Comment:
Does this mean we won't be showing this on production ?

I had the understanding that this would be a feature on top of what the API pulls in

Reply:
I was going on on [this conversion](https://ubxd.slack.com/archives/C010XKV8QLR/p1665417583721359), but seems worth double checking 🤔. I will talk to the design team again tomorrow!